artemis-pkg-copy/src/api/api.toit

```diff
@@ -59,6 +59,9 @@ interface ArtemisService:
   controller-open --mode/int -> int
   static CONTROLLER-OPEN-INDEX /int ::= 6
 
+  controller-open --mode/int --force-recovery-contact/bool -> int
```

Comment:
Not sure we really need another entry for this. Everybody is using the client, so we can just add the default value there.

Reply:
changed the API.